src/style.css

```
@@ -1 +1,16 @@
 /* styles go here */
+.form-field {
+  display: block;
+}
+
+.form-field:not(:last-child) {
+  margin-bottom: 10px;
+}
```

It's hard to understand

```suggestion
.form-field {
  display: block;
  margin-bottom: 10px;
}

.form-field:last-child {
  margin-bottom: 0;
}
```